gst/tensor_decoder/tensordec.h

```diff
@@ -66,6 +66,7 @@ typedef struct
 {
   gint current_label_index; /**< current label index */
   gint new_label_index; /**< new label index */
+  gchar *image_label;       /**< image label that will set to outbuf*/
```

Comment:
The label value changes for every frame. This is a volatile value that doesn't need to be in the class.

Reply:
I will change it as does not use volatile value in class